tests/test_masterclasses.py

```diff
@@ -18,24 +18,28 @@ def test_content_data_category(db, blank_session):
 
 
 @pytest.fixture()
-def new_location(db, blank_session):
+def test_location(db, blank_session):
```

Comment:
In the previous commit, you change one of the fixtures to yield as well. Can you move that change to this commit?

Reply:
Done :)